RNTester/js/examples/PlatformColor/PlatformColorExample.js

```diff
@@ -0,0 +1,258 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const React = require('react');
+const ReactNative = require('react-native');
+import Platform from '../../../../Libraries/Utilities/Platform';
+const {PlatformColor, StyleSheet, Text, View} = ReactNative;
+import {IOSDynamicColor} from '../../../../Libraries/StyleSheet/NativeColorValueTypesIOS';
+import {AndroidColor} from '../../../../Libraries/StyleSheet/NativeColorValueTypesAndroid';
+
+type State = {};
+
+class PlatformColorsExample extends React.Component<{}, State> {
```

Comment:
Can you write these examples with hooks? We are working on migrating all docs examples and RNTester to  be written with hooks so it would be great if these new examples were in that direction too.

Reply:
No problem.   Done